### DISCUSSION TITLE
Include Image Functionality

### DISCUSSION THREAD

**Discussion by keithcrosslin** [excalideck/excalideck#6] [Ideas]
Love this concept!
Are you planning to include the paste image functionality from Excalidraw? When creating decks, I often have an image that I need to use and then paste around it.

**Comment by pscanf** [comment]
Thanks! 🙂 Yes, including that functionality is definitely planned. It's still in beta in Excalidraw, so it's not yet in the `@excalidraw/excalidraw v0.10.0` package Excalideck uses, but I'll work on integrating it when the next version of `@excalidraw/excalidraw` is released.

**Comment by ajdavis** [comment]
Including images in Excalideck would be terrific. I just started using Excalideck but I needed to include an image in my second slide. This is a showstopper so I'm going back to Keynote immediately.

There's no way at all to include images in Excalideck right now, is there?
